packages/neuron-wallet/tests/controllers/sudt.test.ts

```diff
@@ -41,6 +40,7 @@ describe('SUDTController', () => {
     })
 
     it('getSUDTTokenInfo from token list success', async () => {
+      const SUDTController = require("../../src/controllers/sudt").default
```

Comment:
This should be able to `require` once at the top, so you don't need to require multiple times.